iib/workers/tasks/build.py

```diff
@@ -755,14 +752,24 @@ def handle_add_request(
                     ' '.join(excluded_bundles),
                 )
 
-        _opm_index_add(
-            temp_dir,
-            resolved_bundles,
-            prebuild_info['binary_image_resolved'],
-            from_index_resolved,
-            overwrite_from_index_token,
-            (prebuild_info['distribution_scope'] in ['dev', 'stage']),
-        )
+        if is_fbc:
+            opm_registry_add_fbc(
+                base_dir=temp_dir,
```

Comment:
Minor question: Can you help me in understanding your reasoning behind calling a function with named vs positional arguments? For example, here we call with names, whereas in line 765 _opm_index_add we use positional arguments). Even in function opm_registry_add_fbc() there are different ways for calling functions

Reply:
The reason to use names is It happen in the past that somebody modified input parameters of a function but forget to update all its calls. Parameters were passed to wrong names. Since that I prefer calling functions with lot of parameters with their names. I would maybe lean towards making keyword arguments mandatory and not accepting positional ones.

I did not wrote the call of `_opm_index_add`. I just moved it inside `if` statement. That is why it is not used, but I will add it since I am touching the code anyway.

Reply:
Thanks for explaining your thought process.
Added a comment in opm_registry_add_fbc() function